src/main/java/org/folio/circulation/resources/LostItemCirculationRulesEngineResource.java

```diff
@@ -0,0 +1,35 @@
+package org.folio.circulation.resources;
+
+import io.vertx.core.MultiMap;
+import io.vertx.core.http.HttpClient;
+import io.vertx.core.json.JsonArray;
+import org.folio.circulation.domain.Location;
+import org.folio.circulation.rules.Drools;
+import org.folio.circulation.support.Result;
+
+import java.util.concurrent.CompletableFuture;
+
+import static org.folio.circulation.support.Result.succeeded;
+
+
```

Comment:
Please remove this unnecessary blank line

Reply:
Removed